lg_stats/msg/Stats.msg

```diff
@@ -0,0 +1,4 @@
+string lg_system_name
```

Comment:
ok noted.

however, I don't think the topics stipulated on the #126 spec will result in any timeseries data, do they? actually, I wasn't able to locate /activity/active so I am just guessing other other (previous) three.